### PR TITLE
Port some test scripts to Python 3

### PR DESCRIPTION
**Pull request by ernestask**
See commits.

### CODE REVIEW

**Comment by ernestask** [comment]
https://docs.python.org/2/library/basehttpserver.html for reference.

**Comment by ernestask** [comment]
@trapas, shake it.

**Comment by ernestask** [comment]
The pyftpdlib module was copied from https://github.com/giampaolo/pyftpdlib (minus the tests).

**Comment by mkutlak** [comment]
> The pyftpdlib module was copied from [giampaolo/pyftpdlib](https://github.com/giampaolo/pyftpdlib) (minus the tests) (current master, maybe could’ve used a released version instead).

I think it would be better to just use the `python3-pyftpdlib` package for Fedora and then build that package for RHEL in copr.

**Comment by ernestask** [comment]
> I think it would be better to just use the `python3-pyftpdlib` package for Fedora and then build that package for RHEL in copr.

Good idea. Dropped the in-tree copy.

**Comment by mkutlak** [comment]
@trapas retest this please

File: tests/runtests/aux/config.sh [hunk 1]
```diff
@@ -77,5 +77,6 @@ export PACKAGES="abrt \
                  libreport-plugin-logger \
                  libreport-plugin-mantisbt \
                  libreport-plugin-systemd-journal \
+                 python3-pyftpdlib \
```
Comment (mkutlak) [inline tests/runtests/aux/config.sh]: We don't really need the `pyftpdlib` to be available for all tests, just for one.

Can you instead install the package in `rlPhaseStartSetup`? And maybe remove it in `rlPhaseStartCleanup`.

Comment (ernestask) [reply tests/runtests/aux/config.sh]: I added a conditional removal, since blindly removing packages is not nice.

**Comment by mkutlak** [comment]
LGTM. Thank you @ernestask

**Pull request merged**